modules/apps/headless/headless-discovery/headless-discovery-web/src/js/APIGUI.js

```diff
@@ -293,25 +293,29 @@ const APIGUI = () => {
 					</ClayModal>
 				)}
 
-				{!showGraphQL && (
-					<SwaggerUI
-						displayOperationId={true}
-						requestInterceptor={requestInterceptor}
-						supportedSubmitMethods={[
-							'get',
-							'put',
-							'post',
-							'delete',
-							'patch',
-						]}
-						tryItOutEnabled={true}
-						url={
-							endpoint ||
-							endpoints.find((url) =>
-								url.includes('headless-delivery')
-							)
-						}
-					/>
+				{!endpoint.startsWith('http://localhost:8080') ? (
+					<>Error!!</>
```

Comment:
@jaimelr10 please find a way to display a better error. Find the best clay component to do it and talk to beni to decide the message.